src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -146,22 +198,25 @@ trait DataUseRestrictionSupport extends LazyLogging {
     * Create a display-friendly version of the structured data use restriction in the form of a
     * list of code strings.
     *
-    * @param workspace The Workspace
+    * @param attributes The Attributes
     * @return An Attribute Map representing a data use display
     */
-  def generateUseRestrictionDisplayAttribute(workspace: Workspace): Map[AttributeName, Attribute] = {
+  def transformUseRestrictionDisplayAttribute(attributes: Map[AttributeName, Attribute], ontologyDAO: OntologyDAO): Map[AttributeName, Attribute] = {
 
-    val booleanCodes:Seq[String] = getDataUseAttributes(workspace).collect {
+    val booleanCodes:Seq[String] = attributes.collect {
       case (attr: AttributeName, AttributeBoolean(true)) => attr.name
     }.toSeq
 
-    val dsLabels:Seq[String] = (workspace.attributes.get(diseaseLabelsAttributeName) collect {
+
+    val dsLabels:Seq[String] = (attributes.get(AttributeName.withDefaultNS(ConsentCodes.DSURL)) collect {
```

Comment:
This is probably the right thing to do, but I think it needs a corresponding change to `getDiseaseNames` below to not prepend `DataUse.doid_prefix` when calling `OntologyDao.search`.

(GitHub is ridiculous. I originally posted this comment while viewing a subset of commits so it wasn't showing up here. 😡